Yosemite/YosemiteTests/Stores/ReceiptStoreTests.swift

```diff
@@ -415,6 +415,10 @@ final class ReceiptStoreTests: XCTestCase {
     }
 
     func test_generateContent_callsGenerate_passingUnmodified_customerNote() throws {
+        if #available(iOS 18.4, *) {
+            throw XCTSkip("Skipped on iOS 18.4 because of a WebKit crash when running on that OS.")
+        }
+
```

Comment:
Created https://linear.app/a8c/issue/WOOMOB-473 to track this.

See also the dedicated https://github.com/woocommerce/woocommerce-ios/pull/15607 which has more details on the crash.